README.md

```diff
@@ -114,6 +114,27 @@ a working system.
 
 **Note:** If every role does not go green in `pod-status --watch` refer to [Troubleshooting](#troubleshooting)
 
+3. Pulling updates
+
+    When you want to pull the latest changes from the upstream you should:
+
+    ```
+    # Pull the changes (or checkout the commit you want):
+    git pull
+
+    # Update all submodules to match the checked out commit
+    git submodules update --recursive
```

Comment:
Didn't know about `git submodule sync` before but reading the docs now seems weird that we need it ([docs](https://git-scm.com/docs/git-submodule/1.6.5#git-submodule-sync)). The remote urls for our submodules should change that often (if ever). Are you sure this is needed? I would like to add a comment on why this might be the case.